src/css/styles.scss

```diff
@@ -45,6 +45,32 @@ h4 {
   width: max-content;
 }
 
+.recipe--tags {
+  padding: 0px;
+  width: max-content;
+  max-width: 200px;
+  margin-top: 10px;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+
+.recipe--tags span{
+  margin: 0px 2px 2px 0px;
+  color: #359567;
+  background-color: #C4EB67;
+  border-radius: 3px;
+  padding: 1px 4px 1px 4px;
+}
+
+#tag {
```

Comment:
Should this technically be a class? Since it's being used on more than one thing right?